### PR TITLE
[zh-cn] services-networking/service.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -816,6 +818,8 @@ Kubernetes `ServiceTypes` 允许指定你所需要的 Service 类型。
 -->
 * `ClusterIP`：通过集群的内部 IP 暴露服务，选择该值时服务只能够在集群内部访问。
   这也是你没有为服务显式指定 `type` 时使用的默认值。
+  你可以使用向公众暴露服务通过使用[Ingress](/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1/)或者
+  [Gateway API](https://gateway-api.sigs.k8s.io/)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/service.md:822]:
```suggestion
  你可以使用 [Ingress](/zh-cn/docs/reference/kubernetes-api/service-resources/ingress-v1/)
  或者 [Gateway API](https://gateway-api.sigs.k8s.io/) 向公众暴露服务。
```

**Review by windsonsea — COMMENTED** [review]
Only a sentence changed, but show much careless... why?

File: content/zh-cn/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (gaogao101) [reply content/zh-cn/docs/concepts/services-networking/service.md]: Sorry, I will be careful next time. Thanks for the correction.

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -804,6 +804,8 @@ Kubernetes `ServiceTypes` 允许指定你所需要的 Service 类型。
 * `ClusterIP`: Exposes the Service on a cluster-internal IP. Choosing this value
   makes the Service only reachable from within the cluster. This is the
   default that is used if you don't explicitly specify a `type` for a Service.
+  You can expose the service to the public with an [Ingress](docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:807]: The English upstream is incorrect. But we can keep it that way until the upstream has a fix.

**Pull request merged**